### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Keyboard: Keyboard focus is not going to "Manage" button.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2696]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev1; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #AILimited; #DesktopApp; #WCAG2.1.1; #Keyboard; #Win10;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab to all controls and verify keyboard focus is going to "Manage" button or not.

### Actual Result:
Keyboard focus is not going to "Manage" button.

### Expected Result:
Keyboard focus should got to "Manage" button and it should be accessible using keyboard.

### User Impact:
Keyboard users will not perform all operations if keyboard focus is not going to them.

### Attachment for Reference:

[2696_Settings button is not accessible using keyboard.zip](https://github.com/microsoft/vscode-azurefunctions/files/6019934/2696_Settings.button.is.not.accessible.using.keyboard.zip)

**Comment by ejizba** [comment]
This is an upstream issue. Filed https://github.com/microsoft/vscode/issues/120202

**Comment by ejizba** [comment]
Closing in favor of the upstream issue

**Comment by Accessibility56** [comment]
Issue is fixed.
Version: 1.57.0-insider (user setup)
Commit: b95b73b7b074f5dddccd57e5e290617f1a767ec1
Date: 2021-05-05T17:20:18.311Z
Electron: 12.0.4
Chrome: 89.0.4389.114
Node.js: 14.16.0
V8: 8.9.255.24-electron.0
OS: Windows_NT x64 10.0.19042
![image](https://user-images.githubusercontent.com/38523250/117252558-c4b31c80-ae63-11eb-892b-6049630de6d4.png)
